### PR TITLE
Battle log styling

### PR DESCRIPTION
**Pull request by maciekmm**

### CODE REVIEW

File: frontend/src/components/UnitAvatar.tsx [hunk 1]
```diff
@@ -59,6 +65,8 @@ const UnitAvatar: React.FunctionComponent<Props> = ({ unit }: Props) => {
               ${gradientPhases}
             ), 
             conic-gradient(red 0turn, red ${hpPercentage}turn, gray ${hpPercentage}turn, gray 1turn)`,
+        width: `${!!size ? size : 90}px`,
```
Comment (zemiret) [inline frontend/src/components/UnitAvatar.tsx:68]: defaultSize is 80px but here it's 90px? Anyway, it's expanding the gameboard cell when placing a unit, which shouldn't happen

File: frontend/src/components/BattleLog.tsx [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+
+const BattleLog: React.FunctionComponent<Props> = ({
+  log,
+  units,
+  enemyUnits,
+}: Props) => {
   return (
     <div>
-      <p>Battle log</p>
+      <h1 className="caption">Battle log</h1>
```
Comment (zemiret) [inline frontend/src/components/BattleLog.tsx:19]: Use a component `<Caption>`

File: frontend/src/components/UnitAvatar.tsx (hunk 1 above)
Comment (zemiret) [reply frontend/src/components/UnitAvatar.tsx]: It's still expanding the gameboard - by little, but it does.

Comment (maciekmm) [reply frontend/src/components/UnitAvatar.tsx]: Hmm, seems to differ across browsers. In theory there's 72px - 10px (padding) = 62px of space for the avatar. The current height is set to 60px. I further reduced it to 58px and it works fine on chrome now.

**Pull request merged**